{"uuid":"babbe358-8596-409a-87a3-c680697a1176","name":"Scenario Outline: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","children":["871f95a4-b639-4cba-8dc7-b4a6f21c25d0"],"befores":[{"name":"appHooks.Hooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673419178401,"stop":1673419178402}],"afters":[{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673419178410,"stop":1673419178410},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673419178906,"stop":1673419178906},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673419179101,"stop":1673419179101},{"name":"appHooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"6d16f2dd-a533-4e67-8fe4-454d11f2208d-attachment","type":"image/png"},{"name":"Myscreenshot","source":"78cb4452-bd7f-4c39-9271-a2b1e2d26909-attachment"}],"parameters":[],"start":1673419179112,"stop":1673419179413}],"start":1673419178401,"stop":1673419179414}